test/prevent-abbreviations.js

```diff
@@ -43,6 +43,34 @@ const createErrors = message => {
 	return [error];
 };
 
+const anotherNameMessage = 'A more descriptive name will do too.';
+
+const formatMessage = (discouragedName, replacements, nameTypeText, replacementsLimit = 3) => {
```

Comment:
Copy-pasting the tested code (this is literally [`formatMessage` from `rules/prevent-abbreviations.js`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/4c3d61c04318f75480bb6ab30bf948fa1643d550/rules/prevent-abbreviations.js#L363)) diminishes the purpose of testing.

I suggest instead we have a few tests with expected messages hardcoded in the test (at least one for each code path to keep the coverage), and have a bunch of tests without the exact message (with `ruleId` only) to avoid unnecessary repetitiveness.

Reply:
the reason I copied this function is replacements orders really annoying, so I  sort the result. and I  want test the replacements. maybe we can do a unit test for `getNameReplacements`

Reply:
removed